pkgs/development/python-modules/pulumi-aws/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pulumi
+, parver
+, semver
+, isPy27
+}:
+
+buildPythonPackage rec {
+  pname = "pulumi-aws";
+  version = "4.38.0";
+  disabled = isPy27;
```

Comment:
Upstream has no information about this https://github.com/pulumi/pulumi-aws/blob/master/sdk/python/setup.py